packages/shortest/README.md

```diff
@@ -11,22 +11,23 @@ AI-powered natural language end-to-end testing framework.
 
 ## Installation
 ```bash
-npm install shortest
+npm install @antiwork/shortest
 # or
-pnpm add shortest
+pnpm add @antiwork/shortest
 # or
-yarn add shortest
+yarn add @antiwork/shortest
 ```
 
 ## Quick Start
 ```typescript
-import { define } from 'shortest'
+import { UITestBuilder } from '@antiwork/shortest'
```

Comment:
We should not merge this in. This is so ugly! We should fix this first, then update the README (or do in the same PR as that)

Reply:
Sure.